_sources/functions_in_cpp/Exercises.rst

```diff
@@ -0,0 +1,314 @@
+          }
+          return 2 * input + 9;
+        }
+        return input + 4;
+      }  
+
+    - 14
+
+      - Although 14 is divisible by 7, take another look at the conditionals. 
+
+    - 18
+
+      - The flow of code would never reach the last return statement.
+
+    - 36
+
+      - Check your order of operations! 
+
+    - 37
+
+      - Take a closer look at the conditional statements. 
+
+    - 40
+
+      + Since 14 is divisible by 2, the function returns two less than three times 14.
+
+.. mchoice:: functions_in_cpp_mcq_3
+    :practice: T
+
+    If we wanted to create a boolean function called ``isPrime``, which takes an ``int input``
+    as a parameter, which of the following would be the correct function header?
+
+    - ``boolean isPrime (int input) {``
+
+      - In C++, use the ``bool`` keyword for a boolean. 
+
+    - ``bool isPrime (input) {``
+
+      - In a function header, the type of each variable must be specified in the parameter list.
+
+    - ``bool isPrime (int input) {`` 
+
+      + This is the correct function header for the function.
+
+    - ``int isPrime (bool input) {``
+
+      - Take a closer look at what the return type is.
+
+.. mchoice:: functions_in_cpp_mcq_4
+    :practice: T
+
+    If we wrote the following function, which of the other functions below can we also legally write
+    and add to the program?
+
+    .. code-block:: cpp
+
+      int func (double x, bool y);
+
+    - ``int func (double a, bool b);``
+
+      - Since this function has the same name and parameter types as the given function, it is not allowed.
+
+    - ``int foo (double x, bool y);``
+
+      + This function has a different name from the given function, so it is allowed.
+
+    - ``int func (double x);``
+
+      + Although this function has the same name as the given function, it has a different number of parameters, so it is allowed.
+
+    - ``void func (double x, bool y);``
+
+      - Although this function has a different return type, its parameter list is the same as the given function, so it is not allowed.
+
+    - ``int func (bool y, double x);``
+
+      + Although this function has the same name as the given function, its parameter list is in a different order, so it is allowed.
+
+.. mchoice:: functions_in_cpp_mcq_5
+    :practice: T
+
+    What are some benefits of using functions? Select all that are true.
+
+    - Functions make code easier to write and comprehend.
+
+      + You can place replace repetitive code with functions.
+
+    - Functions help encapsulate a set of steps that is repeated multiple times.
+
+      + You can make loops simpler by using functions to do the work.
+
+    - Functions will always return something.
+
+      - ``void`` functions do not return anything.
+
+    - Functions make it easier to detect and fix bugs.
+
+      + Since your code is broken down into many functions, rather than debugging the entire program, you can isolate the function that is causing issues.
```

Comment:
Maybe do this wording instead: "If your code is broken down into many functions,..."